### DISCUSSION TITLE
What's wrong in my font configuration?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#8022] [Q&A]
Today I've updated an ArchLinux system I haven't touched for half a year or so.

Now something is not quite right with fonts in qutebrowser.

Here are a few screenshot showing what I mean by "wrong", plus some comments:

![font-sizes-on-teams](https://github.com/qutebrowser/qutebrowser/assets/20521900/5220d339-f246-4fde-b216-56e5f611fcce)
![font-sizes-on-facebook](https://github.com/qutebrowser/qutebrowser/assets/20521900/b04a0d33-c350-414f-af1b-5daf0b37cf98)
![font-sizes-on-outlook](https://github.com/qutebrowser/qutebrowser/assets/20521900/86071de8-346b-4c92-928c-6654626cb6ae)
![font-sizes-on-settings](https://github.com/qutebrowser/qutebrowser/assets/20521900/42b07c2d-1845-4267-8e77-e9990152e10a)

[Here's my config.py](https://gist.github.com/Aster89/7c9895dbc32a2f889cc40614c4893a3e), and [here's my autoconfig.yml](https://gist.github.com/Aster89/6ffced168911b9266d3d2378fc35efbd), which contains redundant stuff (I mean, stuff which is overridden anyway by config.py).

I've played around with the settings a bit, but haven't really got anywhere.

Any help is appreciated.

**Comment by Aster89** [comment]
I wonder if this just means that the websites are done like shit and hardcode `px`-sized fonts, in which case I should probably just set back the values of the font options to lower numbers like `12pt` for `default_size` and then rely on `zoom.default` to make everything bigger.

**Comment by The-Compiler** [comment]
At least for me, I saw something similar in various applications due to a [Nerd Font naming change](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.1.0). Though that doesn't seem to mention DejaVu Sans Mono. Still, if it happened with a recent upgrade, I'd suspect that.
